bubble-wrap-style.yaml

```diff
@@ -2342,7 +2224,7 @@ layers:
                             }
                         }
                     sprite_default: generic_shield_5char
-                    size: [[7,[90%,90%]],[12,[100%,100%]],[15,[110%,110%]]]
+                    size: [[7,[75%,75%]],[12,[100%,100%]],[15,[110%,110%]]]
```

Comment:
Minor note, with %-based sprite scaling you should be able to write this with a single value since it's the same in both dimensions, e.g. 

`size: [[7, 75%],[12, 100%],[15, 110%]]`